controllers/upgrade_handlers.go

```diff
@@ -477,6 +477,14 @@ func (u *UpgHandler) HandleBackup(ctx context.Context, ibu *lcav1alpha1.ImageBas
 	// trigger and track each group
 	for index, backups := range sortedBackupGroups {
 		u.Log.Info("Processing backup", "groupIndex", index+1, "totalGroups", len(sortedBackupGroups))
+
+		// check for any stale backup in the group
+		if allStaleBackupsRemoved, err := u.BackupRestore.CleanupStaleBackups(ctx, backups); err != nil {
```

Comment:
The function CleanupStaleBackups was changed in [PR404](https://github.com/openshift-kni/lifecycle-agent/pull/404/files) to return only one variable.